scripts/kconfig/kconfig.py

```diff
@@ -182,6 +185,32 @@ def promptless(sym):
 
     return not any(node.prompt for node in sym.nodes)
 
+def write_kconfig_filenames(paths, root_path, output_file_path):
+    # 'paths' is a list of paths. The list has duplicates and the
+    # paths are either absolute or relative to 'root_path'.
+
+    # We need to write this list, in a format that CMake can easily
+    # parse, to the output file at 'output_file_path'.
+
+    # The written list should also have absolute paths instead of
+    # relative paths, and it should not have duplicates.
+
+    # Remove duplicates
+    paths_uniq = set(paths)
+
+    with open(output_file_path, 'w') as out:
+        for path in paths_uniq:
```

Comment:
I think iterating through a `set()` gives a non-deterministic order, so might want to do

```python
for path in sorted(set(paths)):
    ....
```

Makes it easier to compare versions of the file.

Reply:
Good catch, fixed.